### PR TITLE
Update tar.gz, zip infra-agent installation with checksums verification

### PR DESCRIPTION
**Pull request by brushknight**
Please follow [conventional commit standards](https://www.conventionalcommits.org/en/v1.0.0/)
in your commit messages and pull request title.

## Give us some context

* Add docs on how to verify checksums for tar.gz and zip

- [ ] Add the same for manual zip, tar.gz installation

### CODE REVIEW

File: src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux.mdx [hunk 1]
```diff
@@ -24,9 +24,13 @@ Before installation, check the [compatibility and requirements](/docs/infrastruc
 
 To install the agent:
 
-1. Download the [packaged agent file](https://download.newrelic.com/infrastructure_agent/binaries/linux/).
-2. Unpack the file.
-3. Make sure the file unpacks with the following structure:
+1. Download the [packaged agent file](https://download.newrelic.com/infrastructure_agent/binaries/linux/) or use our one-liner command to fetch the given version of the agent with checksum and automatically verify it. You will need to set desired architecture (amd64, 386, arm64, arm) and [version](https://github.com/newrelic/infrastructure-agent/releases)
```
Comment (josemore) [inline src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux.mdx:27]:
```suggestion
1. Download the [packaged agent file](https://download.newrelic.com/infrastructure_agent/binaries/linux/) or use the following command that automatically fetches a specific version of the agent, its checksum and verifies it after download. Replace `ARCH=amd64` with desired architecture (amd64, 386, arm64, arm) and `V=1.127.3` with [latest or specific version](https://github.com/newrelic/infrastructure-agent/releases/latest). 
```

File: src/content/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-assisted-install-infrastructure-agent-windows.mdx [hunk 2]
```diff
@@ -22,9 +22,21 @@ With the assisted install of the infrastructure agent for Windows, you can make
...
 
-1. Download the [packaged agent file](https://download.newrelic.com/infrastructure_agent/binaries/windows/).
-2. Unpack the file.
-3. Make sure the file unpacks with the following structure:
+1. Download the [packaged agent file](https://download.newrelic.com/infrastructure_agent/binaries/windows/) or use our one-liner command to fetch the latest version of the agent with checksum and automatically verify it.
+
+   **amd64**
+
+   ```
+   $url="https://download.newrelic.com/infrastructure_agent/windows/newrelic-infra.msi";@("$url", "$url.sum") | ForEach-Object { Invoke-WebRequest -Uri $_ -OutFile $_.Split('/')[-1] }; write-host 'Checksum:' $(If (Select-String -Path "$url.sum".Split('/')[-1] -Pattern (Get-FileHash $url.Split('/')[-1]).Hash -Quiet) {"Ok"} Else {"Fail"})
```
Comment (brushknight) [inline src/content/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-assisted-install-infrastructure-agent-windows.mdx:30]: Just noticed that it is about msi and not zip, I will update it.

File: src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux.mdx (hunk 1 above)
Comment (brushknight) [reply src/content/docs/infrastructure/install-infrastructure-agent/linux-installation/tarball-assisted-install-infrastructure-agent-linux.mdx]: updated

File: src/content/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-assisted-install-infrastructure-agent-windows.mdx (hunk 2 above)
Comment (brushknight) [reply src/content/docs/infrastructure/install-infrastructure-agent/windows-installation/zip-assisted-install-infrastructure-agent-windows.mdx]: fixed

**Comment by brushknight** [comment]
Before making this a PR we will need to test the solution with the production bucket.

**Review by x8a — APPROVED** [review]

**Pull request merged**